### ISSUE TITLE
First line indentation within syntax highlight

### ISSUE THREAD

**Issue by alves-gabriel** [zerodevx/zero-md#67]
How can I avoid indentation with zero-md? 

I have included a .md file to my github page using zero-md. The rendering however introduces indentation to the file (including to the syntax highlighted blocks). Is there any way to avoid this?

![image](https://user-images.githubusercontent.com/42458274/156907625-29584999-c76b-4246-bed5-55d869a12738.png)

Thank you!

**Comment by zerodevx** [comment]
It could be that your MD file is malformed. Can you show the contents here?

**Comment by alves-gabriel** [comment]
Thank for your reply! Of course! The relevant part of the markdown code is as follows (this is correctly rendered in github, without the indentation):

    # Foldr examples
    
    ### Multiplying two number using (+)
    
    Multiplication between two numbers n and m can be very naturally implemented with ```foldr``` (only for the sake of the example, of course). We can i) build a function which constructs an array with m copies of a number n and ii) apply foldr and the (+) operator to this array.
    
    ```haskell
    Prelude> let mcopy m n | m < 0 = [] | otherwise = n:mcopy (m-1) n
    Prelude> mult m n = foldr (+) 0 (mcopy m n)
    Prelude> mcopy 5 2
    [2,2,2,2,2,2]
    Prelude> mult 5 2
    12
    ```
    Note how the list is constructed _recursively_.

Thank you for your time.

**Comment by zerodevx** [comment]
Not sure what's wrong tbh. Can you post your HTML code? Or better yet, if you could give a minimal repro with [plunkr](https://plnkr.co/) it'll be easier to help.

**Comment by alves-gabriel** [comment]
Hi, @zerodevx! Everything is solved now! 

![image](https://user-images.githubusercontent.com/42458274/156995123-a051ce06-3973-45bf-ad69-120604c1ebac.png)

I discovered the problem when constructing a minimal repro with plunkr (thank you for the link by the way)! Unfortunately I had two CSS classes with conflicting definitions (one of them was internal). Namely, I had the following class in my main.css:

    .tab {
      text-indent:40px;
      text-align: justify;
    }

And an internal one (with same name) for anchoring some sections in the page. This is embarrassing, I'm not sure how this escaped me! I did not know the external .css files would affect the markdown as well. Sorry for inconveniencing you with such a stupid mistake and thanks a lot for your time. Your project has been working wonders for me! :D
